template/config/meta/package.json

```diff
@@ -43,7 +48,17 @@
     "tailwindcss": "^1.4.1",
     "typescript": "^3.0.0",
     "vue-template-compiler": "^2.5.21",
-    "webpack-bundle-analyzer": "^3.7.0"
+    "webpack-bundle-analyzer": "^3.7.0",
+    "@storybook/addon-links": "^5.3.14",
+    "@storybook/addon-options": "^5.3.14",
+    "@storybook/preset-scss": "^1.0.2",
+    "@storybook/preset-typescript": "^2.1.0",
+    "@storybook/vue": "^5.3.19",
+    "@types/storybook__vue": "^5.0.1",
+        "babel-preset-vue": "^2.0.2",
+        "npm-run-all": "^4.1.5"
```

Comment:
Every new dependency here **MUST** be defined as part of the storybook manifest.json